pkg/util/membercluster_client.go

```diff
@@ -20,12 +20,15 @@ import (
 )
 
 const (
-	// kubeAPIQPS is the maximum QPS to the master from this client
-	kubeAPIQPS = 20.0
-	// kubeAPIBurst is the maximum burst for throttle
-	kubeAPIBurst = 30
-	tokenKey     = "token"
-	cADataKey    = "caBundle"
+	tokenKey  = "token"
+	cADataKey = "caBundle"
+)
+
+var (
+	// KubeAPIQPS is the maximum QPS to the master from this client
+	KubeAPIQPS float32 = 20.0
+	// KubeAPIBurst is the maximum burst for throttle
+	KubeAPIBurst int = 30
```

Comment:
It eventually degenerates to a global variable from a const. (I don't like the `const` as well, just don't have time to improve it).
Let's see if there is a better solution for this.

Reply:
I talked with @mrlihanbo yesterday, he will solve this issue before moving this PR forward.

Reply:
Glad to hear that. I will rebase this PR after everything is ok.

Reply:
You can start your work now.